atomicapp/nulecule/base.py

```diff
@@ -237,7 +237,7 @@ def load(self, nodeps=False, dryrun=False):
         """
         Load external application of the Nulecule component.
         """
-        if not self.artifacts:
+        if self.source:
```

Comment:
``source`` contains the value of the ``source`` attr of a graph item. I felt that checking for ``self.source`` makes mores sense than a negative check on ``self.artifacts``.